deltachat-ffi/src/lib.rs

```diff
@@ -710,6 +710,25 @@ pub unsafe extern "C" fn dc_send_text_msg(
     })
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn dc_send_videochat_invitation(
+    context: *mut dc_context_t,
+    chat_id: u32,
+) -> u32 {
+    if context.is_null() {
+        eprintln!("ignoring careless call to dc_send_videochat_invitation()");
+        return 0;
+    }
+    let ctx = &*context;
+
+    block_on(async move {
+        chat::send_videochat_invitation(&ctx, ChatId::new(chat_id))
+            .await
+            .map(|msg_id| msg_id.to_u32())
+            .unwrap_or_log_default(&ctx, "Failed to send videochat invitation")
```

Comment:
You should avoid this function (and we should remove it completely).  It invokes `error!` and this is almost certainly the wrong thing to do here.  If an error toast was meant to be displayed the rust lib should have done this explicitly.

You should use `.log_err(&ctx).unwrap_or_default()` instead.

Reply:
well, i just did copy+paste for a similar function :) 

form the ui-point of view, an `error!` that is shown to the user in this case is completely fine.

if we want to move the `error!` out of the ffi, we maybe should better do that in a separate pr/issue. in general, i think it is not the worse when errors bubble up up to the ffi and reported as error! from there (as the ffi currently does not have other good error handling). but as said, maybe discuss that outside this pr.